rest/blip_client_test.go

```diff
@@ -144,23 +153,88 @@ func (btr *BlipTesterReplicator) initHandlers(btc *BlipTesterClient) {
 				panic(err)
 			}
 
-			var old db.Body
 			btc.docsLock.RLock()
 			oldBytes := btc.docs[docID][deltaSrc]
 			btc.docsLock.RUnlock()
-			if err := old.Unmarshal(oldBytes); err != nil {
+			if err := bodyJSON.Unmarshal(oldBytes); err != nil {
```

Comment:
Failing test suggests I've broken something around here. Will fix and let you know when.